src/install/mongodb/appInfo.mdx

```diff
@@ -0,0 +1,37 @@
+---
+componentType: appInfoConfig
+headingText: Choose your framework
+---
+
+<Callout variant="tip">
+
+**Use guided install to quickly see your data in the UI**
+
+Guided install is a single CLI command you can run on your system to monitor your MongoDB instance. The CLI installs the infrastructure agent and MongoDB integration.
+
+This is a good way to trial ingesting data on our platform. 
```

Comment:
This should be moved up before starting the in-doc steps. As a reader, I'm wondering if the guided install link changes based on my first three selections in the "Choose a framework" section.